src/styles/button.css

```diff
@@ -0,0 +1,36 @@
+.product-card__buy-button {
+  display: flex;
+  position: absolute;
+  left: 16px;
+  right: 16px;
+  top: 348px;
+  bottom: 16px;
+  width: 168px;
+  height: 40px;
+  justify-content: center;
+  align-items: center;
+  border-radius: 5px;
+  background: #00acdc;
+}
+
+.button {
+  text-decoration: none;
+  color: #fff;
+  width: 168px;
+  height: 40px;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 40px;
+  text-align: center;
+}
+
+.buy-button__link--active:hover {
+  text-decoration: none;
+  background-color: #fff;
+  color: #00acdc;
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+  width: 168px;
+  height: 40px;
+  line-height: 40px;
+}
```

Comment:
You don't need to include styles that don't change from the original condition to :hover pseudoclass.